docs/access_rights.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Access rights
-description: Documentation of access rights and publication on the Apify platform.
+description: Control permissions for private resources such as actors, actor runs and storages. Allow other users to share, modify or build new versions.
```

Comment:
I'd update this to:

... read, ~share~ run, modify, or build new versions.

as this is allowed by sharing.